pkg/registry/pod/rest/log.go

```diff
@@ -58,6 +58,9 @@ func (r *LogREST) Get(ctx api.Context, name string, opts runtime.Object) (runtim
 	if err != nil {
 		return nil, err
 	}
+	if location.Host == "" {
```

Comment:
IIRC it was for debugging at some point, but I figured out that they don't do any harm. I removed them now.